CountryPickeriOS13+/CountryPickerSPM/Sources/CountryPickerSPM/Country.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Country.swift
+//  CountryCodeInSwift3
+//
+//  Created by Cl-macmini-100 on 12/19/16.
+//  Copyright © 2016 Suryakant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+open class Country: Identifiable {
```

Comment:
Why open?

Reply:
We need to expose `Country` to client. they can get country in callback